makerspace/templates/tools/admin_tool.html

```diff
@@ -0,0 +1,40 @@
+{% extends "web/base.html" %}
+{% load static %}
+{% load i18n %}
+
+{% block body %}
+    <h1 class="ui text-centered make_col_yellow header">
+        {% trans "Tools" %}
+    </h1>
+    <div class="ui text container">
+        <a class="ui yellow button" href="{% url 'makerspace-tools-create' %}">
+            <i class="ui add icon green"></i>
```

Comment:
The green color here is not used in these buttons elsewhere. Should be removed for consistency.

Maybe even remove the button entirely and use a plus icon by the "Tools" title similarly as for machines.